### PR TITLE
Show car on voucher head for weekly sale vouchers

### LINKED ISSUE #230: Car not showing on voucher head for weekly sale vouchers
When a weekly sale has an account selected with car enabled and we create a new weekly sale journal all the vouchers in that journal get the car id in the database but only on the voucher head the car is not shown.

### PR DESCRIPTION
**Pull request by Baxxx**
Close #230

### CODE REVIEW

**Comment by Baxxx** [comment]
Also changed the order of tables from `Cars, Departments, Projects` to `Departments, Projects, Cars` on journal edit, weekly sale template and weekly sale edit pages, and added row for cars on weekly sale edit page.

**Comment by sls-lnluksa** [comment]
@stoivo Alignment of car columns on weeklysale edit page is different for sale table and credit table.
Check if it should be the same.

First round review: Nikola Lukovic (sls-lnluksa)
Final review with: @stoivo

**Comment by Baxxx** [comment]
Should `Mengde` on journal edit page be paired with car column?
On weekly sale edit page, if the selected `CarID` is 0, print `Diverse`

File: modules/weeklysale/view/edit.php [hunk 1]
```diff
@@ -13,6 +13,22 @@
...
 
+$query_sale = "select * from weeklysalegroupconf where WeeklySaleConfID = '".$weeklysale->head->WeeklySaleConfID."' order by Type";
+$weekly_sales_groups_result_set = $_lib['db']->db_query($query_sale);
+
+// The query above selects two rows with configurations for all 20 groups
+// where first row is for sales and the second row is for credit
+$sales_group_conf_object = $_lib['db']->db_fetch_object($weekly_sales_groups_result_set);
+$credit_group_conf_object = $_lib['db']->db_fetch_object($weekly_sales_groups_result_set);
+
+$car_query = "select * from car";
```
Comment (Baxxx) [inline modules/weeklysale/view/edit.php]: Only get active ones.

**Pull request merged**